cpp/algo_module/algorithm/algo.hpp

```diff
@@ -1,5 +1,220 @@
 #pragma once
 
+#include <cmath>
+#include <memory>
 #include <mgp.hpp>
+#include <queue>
+#include <unordered_map>
+#include <unordered_set>
+#include <vector>
 
-namespace Algo {}  // namespace Algo
+namespace Algo {
```

Comment:
Since we have all these stuff for astar, does it make sense to add aditional namespace for AStar?

Reply:
Dunno, maybe, lets discuss it on daily?

Reply:
Don't have to, resolved